tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "skipLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "@deck.gl-community/bing-maps": ["modules/bing-maps/src"],
```

Comment:
This module has never been published. Would it not be better to use `@deck.gl-community/bing-maps` and update `modules/bing-maps/package.json`?

Reply:
Done! ✅